packages/round-manager/src/features/round/ViewRoundStats.tsx

```diff
@@ -57,29 +64,27 @@ export default function ViewRoundStats() {
         <div className={"mr-10 flex items-center "}>Overview</div>
         <StatsCard
           text={
-            roundStats
-              ? new Intl.NumberFormat("en-US", {
-                  style: "currency",
-                  currency: "USD",
-                }).format(roundStats.amountUSD)
-              : "-"
+            round &&
+            new Intl.NumberFormat("en-US", {
+              style: "currency",
+              currency: "USD",
+            }).format(round.amountUSD)
           }
           title={"Est. Donations Made"}
         />
         <StatsCard
-          text={matchAmount ? matchAmount[0] : "-"}
+          text={
+            matchAmount &&
```

Comment:
one thing to note here that may be confusing to the user: is this token amount, or USD amount displayed in the UI? We should consider having this displayed.